ansible/roles/docker/templates/ubuntu1804_sharedlibs.Dockerfile.j2

```diff
@@ -58,7 +58,7 @@ ENV OPENSSL300DIR /opt/openssl-3.0.0
 
 RUN mkdir -p /tmp/openssl_3.0.0 && \
     cd /tmp/openssl_3.0.0 && \
-    curl -sL https://www.openssl.org/source/openssl-3.0.0-alpha14.tar.gz | tar zxv --strip=1 && \
+    git clone git@github.com:quictls/openssl.git && cd openssl && \
```

Comment:
This doesn't work as the docker image has no git credentials. Using a https://github.com/... URL would probably work.

Reply:
```suggestion
    git clone https://github.com/quictls/openssl.git -b openssl-3.0.0-alpha15+quic --depth 1 && \
    cd openssl && \
```
@danbev Do you think we should test against both the quictls fork and non-forked versions of OpenSSL (instead of just replacing the non-forked with the fork)?